vault/overlays/nerc-ocp-infra/backup-job/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+generatorOptions:
+  disableNameSuffixHash: true
+namespace: vault
+configMapGenerator:
+  - files:
+      - taskrun.yaml
+    name: backup-vault-run
+resources:
```

Comment:
Are `toolkitpod.yaml` and `taskrun.yaml` also to be applied manually alongside `secret.yaml`?

Reply:
@knikolla `taskrun.yaml` is deployed as a `ConfigMap` (which is mounted in the `trigger-backup-job` `CronJob`.

Reply:
Ah I see. I missed the `ConfigMap` generator 2 lines above.

And I see the `toolkitpod.yaml` file is heavily commented about its usage. It can get confusing when some files in the same dir and file extensions are meant to be used differently.